lib/releaselib.sh

```diff
@@ -911,17 +911,13 @@ release::docker::release () {
     "hyperkube"
   )
 
-  if [[ -n $G_AUTH_USER && $registry == "gcr.io/google_containers" ]];then
+  if [[ -n "$G_AUTH_USER" && "$registry" == "$GCRIO_PATH_PROD" ]]; then
     logrun $GCLOUD config set account $G_AUTH_USER
   fi
 
   if [[ -d "$release_images" ]]; then
     release::docker::release_from_tarfiles $* || ret=$?
   else
-    # TODO: remove this when all kubernetes releases produce a release-images
```

Comment:
I think this TODO meant the entire else block could be removed. (I think all releases produce a release-images directory now?)